lib/v2/screens/explore_screens/vote_screens/proposals/components/voting_end_cycle_card.dart

```diff
@@ -20,79 +20,93 @@ class VotingCycleEndCard implements SliverPersistentHeaderDelegate {
           padding: const EdgeInsets.all(16.0),
           child: Container(
             decoration: BoxDecoration(color: AppColors.darkGreen2, borderRadius: BorderRadius.circular(12)),
-            child: Padding(
-              padding: const EdgeInsets.all(16.0),
-              child: SingleChildScrollView(
-                physics: const BouncingScrollPhysics(),
-                child: Column(
-                  mainAxisSize: MainAxisSize.min,
-                  crossAxisAlignment: CrossAxisAlignment.start,
-                  children: [
-                    Text('Voting cycle ends in'.i18n),
-                    const SizedBox(height: 16.0),
-                    BlocBuilder<VoteBloc, VoteState>(
-                      builder: (context, state) {
-                        switch (state.pageState) {
-                          case PageState.initial:
-                            return const SizedBox.shrink();
-                          case PageState.loading:
-                            return Row(
-                              mainAxisAlignment: MainAxisAlignment.center,
-                              children: [const SizedBox(height: 20, width: 20, child: CircularProgressIndicator())],
-                            );
-                          case PageState.failure:
-                            return Row(
-                              mainAxisAlignment: MainAxisAlignment.center,
-                              children: [Text(state.errorMessage!, style: Theme.of(context).textTheme.headline5)],
-                            );
-                          case PageState.success:
-                            return Row(
-                              children: [
-                                Expanded(
-                                  child: Row(
+            child: SingleChildScrollView(
+              padding: const EdgeInsets.only(top: 16, bottom: 16, right: 14, left: 14),
```

Comment:
I did'nt know the SingleChildScrollView has a padding parameter.

Reply:
> I did'nt know the SingleChildScrollView has a padding parameter.

I think they added this on flutter 2.0